test/testDataCache.js

```diff
@@ -47,7 +47,7 @@ export const testDataCache = {
         test.expect(1);
         let cache = DataCache.getDataCache();
 
-        const data = cache.getData("basename", new Locale("en-US"));
+        const data = cache.getData("root", "basename", new Locale("en-US"));
```

Comment:
I thought it would be the real path that we could call 'root` location. Does take put the string 'root'?

Reply:
The first parameter is a string that is a path to a root directory like "/a/b/c" or "../x" or just "root". I just used "root" because it was simple and it just had to be a string that was the same when you put the data and you get the data.

Reply:
oh yes, it's just a sample. path info will come as a type string.